crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -49,6 +49,7 @@ mod metrics;
 pub(crate) mod tests;
 
 const TASKS_PER_CORE: usize = 1;
+const END_OF_EPOCH_BROADCAST_CHANNEL_CAPACITY: usize = 5;
```

Comment:
curious why 5?

Reply:
Just threw something in there for now. Should probably be 1 or 2